FFXIVClientStructs/FFXIV/Component/GUI/AtkComponentList.cs

```diff
@@ -22,9 +22,50 @@ public unsafe struct AtkComponentList {
     [FieldOffset(0x148)] public AtkCollisionNode* HoveredItemCollisionNode;
     [FieldOffset(0x150)] public int HoveredItemIndex2; // Repeat?
     [FieldOffset(0x158)] public int HoveredItemIndex3; // Repeat?
+    [FieldOffset(0x193)] public bool IsUpdatePending;
+
+    [VirtualFunction(25)]
+    public partial AtkComponentListItemRenderer* GetItemRenderer(int index);
+
+    [VirtualFunction(26)]
+    public partial void SetItemDisabledState(int index, bool disabled);
+
+    [VirtualFunction(27)]
+    public partial bool GetItemDisabledState(int index);
+
+    [VirtualFunction(28)]
+    public partial void SetItemHighlightedState(int index, bool highlighted, bool triggerUpdate = true);
+
+    [VirtualFunction(29)]
+    public partial bool GetItemHighlightedState(int index);
+
+    [VirtualFunction(31)]
+    public partial void SelectItem(int index, bool dispatchEvent37 = false);
+
+    [VirtualFunction(32)]
+    public partial void DeselectItem();
```

Comment:
In cases like the ContentsFinder, does this deselect the last selected item? all items?

Reply:
This the duty selection in ContentsFinder is unrelated to the selected item in the list.
See how it changes the SelectedItemIndex when I first click on an item:

https://github.com/aers/FFXIVClientStructs/assets/96642047/e7f31a5e-868a-4a29-ac04-ebff2d54b6d4

When I call DeselectItem, it removes the highlight on the item and sets SelectedItemIndex to -1.

Reply:
Ahh I see, so it's just the yellow selection background, which only applies to one item at a time. Thanks for the clarification!